### ISSUE TITLE
windows: file-path in lib\cmake\PySide-1.2.4\PySideConfig-python3.5.cmake have wrong seperator

### ISSUE THREAD

**Issue by looooo** [conda-forge/pyside-feedstock#26]
The CMake of FreeCAD stops at the pyside configuration step because some paths can't be read. I think this is because of wrong seperator type.

**Comment by looooo** [comment]
```
if(MSVC)
    SET(PYSIDE_LIBRARY "C:/bld/recipe_1485457064061/_b_env/Library/lib/pyside-python3.6.lib")
elseif(CYGWIN)
    SET(PYSIDE_LIBRARY "C:/bld/recipe_1485457064061/_b_env/Library/lib/pyside-python3.6.lib")
elseif(WIN32)
    SET(PYSIDE_LIBRARY "C:/bld/recipe_1485457064061/_b_env/Library/bin/pyside-python3.6.dll")
else()
    SET(PYSIDE_LIBRARY "C:/bld/recipe_1485457064061/_b_env/Library/lib/pyside-python3.6.dll")
endif()
SET(PYSIDE_PYTHONPATH "C:\bld\recipe_1485457064061\_b_env\Lib\site-packages")
SET(PYSIDE_TYPESYSTEMS "C:/bld/recipe_1485457064061/_b_env/Library/share/PySide/typesystems")
````
The problem is in PySideConfig-python3.5.cmake.

**Comment by jschueller** [comment]
what's the error message/traceback ?

**Comment by jschueller** [comment]
I found a fix in #27, but other problems showed up

**Comment by looooo** [comment]
thanks!
